dftimewolf/lib/collectors/grr_hosts.py

```diff
@@ -328,6 +332,11 @@ def _ProcessThread(self, client):
         ignore_interpolation_errors=True,
         apply_parsers=False)
     flow_id = self._LaunchFlow(client, 'ArtifactCollectorFlow', flow_args)
+    if not flow_id:
+      msg = 'Flow could not be launched on {0:s}.'.format(client.client_id)
+      msg += '\nArtifactCollectorFlow args: {0:s}'.format(flow_args)
```

Comment:
Do you need this format string to be '{0!s}' since flow_args isn't actually a string?

Reply:
Correct!